DeepSpeech.py

```diff
@@ -723,6 +725,13 @@ def do_graph_freeze(output_file=None, output_node_names=None, variables_blacklis
                 variable_names_blacklist=variables_blacklist,
                 initializer_nodes='')
 
+            input_node_names = []
+            return strip_unused_lib.strip_unused(
+                input_graph_def=frozen,
+                input_node_names=input_node_names,
+                output_node_names=output_node_names.split(','),
+                placeholder_type_enum=dtypes.float32.as_datatype_enum)
```

Comment:
You shouldn't need to go through `tensorflow.dtypes.float32`, just `tf.float32` should be enough.